code/datums/spells/summonitem.dm

```diff
@@ -24,7 +24,7 @@
 		if(!marked_item) //linking item to the spell
 			message = "<span class='notice'>"
 			for(var/obj/item in hand_items)
-				if(istype(item, /obj/item/organ/brain)) //Yeah, sadly this doesn't work due to the organ system.
+				if(istype(item, /obj/item/organ/internal/brain)) //Yeah, sadly this doesn't work due to the organ system.
```

Comment:
note to self: make this work with this new organ system.  Ripping someone's brain out would be hilarious.